packages/runtime/test/consumption/attributes.test.ts

```diff
@@ -1258,6 +1331,40 @@ describe(SucceedRelationshipAttributeAndNotifyPeerUseCase.name, () => {
     });
 });
 
+describe(ChangeDefaultRepositoryAttributeUseCase.name, () => {
+    test("should change default repository attribute", async () => {
+        const defaultAttribute = (
+            await services1.consumption.attributes.createRepositoryAttribute({
+                content: {
+                    value: {
+                        "@type": "GivenName",
+                        value: "My default name"
+                    }
+                }
+            })
+        ).value;
+
+        const desiredDefaultAttribute = (
+            await services1.consumption.attributes.createRepositoryAttribute({
+                content: {
+                    value: {
+                        "@type": "GivenName",
+                        value: "My new default name"
+                    }
+                }
+            })
+        ).value;
+
+        const result = await services1.consumption.attributes.changeDefaultRepositoryAttribute({ attributeId: desiredDefaultAttribute.id });
+        expect(result.isSuccess).toBe(true);
+        const newDefaultAttribute = result.value;
+        expect(newDefaultAttribute.default).toBe(true);
+
+        const updatedFormerDefaultAttribute = (await services1.consumption.attributes.getAttribute({ id: defaultAttribute.id })).value;
+        expect(updatedFormerDefaultAttribute.default).toBeUndefined();
+    });
+});
+
```

Comment:
Perhaps you could combine these two test in  one and if you want add a test, that if someone deletes the default attribute one of the remaining attributes of the same type will be the default.

Reply:
Which two tests do you mean?

Regarding adding a test that checks if the default is transmitted correctly, I already do so in consumption. I think this should principally be sufficient.

Reply:
"should create a RepositoryAttribute that is the default if it is the first of its value type"
and 
"should create a RepositoryAttribute that is not the default if it is not the first of its value type" these two tests.

Reply:
It is good practice to check one thing per test. Thus, I'd like to keep them separated.